### PR TITLE
Update the Samples (v6)

### PR DESCRIPTION
**Pull request by lipchev**
- projects converted to Sdk style
- added a new Directory.Packages for the Samples folder and updated the references to the latest (working) versions
- minor fixes to the UnitConverter.Wpf

### CODE REVIEW

**Comment by lipchev** [comment]
I've been experimenting (locally) with the following trick:
1. I've added another `Nuget.config` with two package sources: one for `https://api.nuget.org/v3/index.json` and one for `../Artifacts/NuGet`
2. In the project properties I add a condition (for now this is simply `$(Configuration)' != 'Release'` ) in which I override the UnitsNet package version: `<PackageReference Include="UnitsNet" VersionOverride="6.0.0-pre012" />`
3. Building the main solution using the build script generates the nuget with it's pre-release version which is then picked up by the Samples projects that have these conditions (I'm not sure how to avoid hard-coding the pre-release version number)

I'm not sure if this is going to work on the build server, so after this is merged- I thought I'd give it a try and see what happens.. Ideally, along with the _godzilla PR_  I was hoping to introduce a new project in the samples called `UnitsNetSetup.Configuration`, targeting the pre-release nuget (that should hopefully get build on the server), in which I showcase the new features, regarding the different configuration options, extending with custom units etc..

File: Samples/MvvmSample.Wpf/MvvmSample.Wpf/MvvmSample.Wpf.csproj [hunk 1]
```diff
@@ -1,163 +1,29 @@
...
-      <HintPath>..\..\packages\Prism.Core.7.1.0.431\lib\net45\Prism.dll</HintPath>
-    </Reference>
-    <Reference Include="Prism.Unity.Wpf, Version=7.1.0.431, Culture=neutral, PublicKeyToken=40ee6c3a2184dc59, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Prism.Unity.7.1.0.431\lib\net45\Prism.Unity.Wpf.dll</HintPath>
-    </Reference>
-    <Reference Include="Prism.Wpf, Version=7.1.0.431, Culture=neutral, PublicKeyToken=40ee6c3a2184dc59, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\Prism.Wpf.7.1.0.431\lib\net45\Prism.Wpf.dll</HintPath>
-    </Reference>
-    <Reference Include="System" />
+    <!-- managing these dependencies is a nightmare   -->
```
[the remark this answers is not in the document]
Comment (angularsen) [reply Samples/MvvmSample.Wpf/MvvmSample.Wpf/MvvmSample.Wpf.csproj]: Okay I think I figured out a solution where we can upgrade all nugets and fix the dependency issue, see 0f552b9e8acae49409b6eaf7b1f2e17ca0590204.

Also added solution files for scripts and .props file.

On a related note, fuck WPF and particularly fuck XAML. What a trainwreck to work with.

**Review by angularsen — APPROVED** [review]

File: Samples/MvvmSample.Wpf/MvvmSample.Wpf/MvvmSample.Wpf.csproj (hunk 1 above)
[the remark this answers is not in the document]
Comment (lipchev) [reply Samples/MvvmSample.Wpf/MvvmSample.Wpf/MvvmSample.Wpf.csproj]: Oh, come on.. I love WPF. They just need to update the XAML to the latest specs, and fix those 20 year old issues with CollectionChanged and what not.. 😄

[the remark this answers is not in the document]
Comment (lipchev) [reply Samples/MvvmSample.Wpf/MvvmSample.Wpf/MvvmSample.Wpf.csproj]: ..and we'll be good for another 20 years..

[the remark this answers is not in the document]
Comment (angularsen) [reply Samples/MvvmSample.Wpf/MvvmSample.Wpf/MvvmSample.Wpf.csproj]: Nooo

**Pull request merged**